bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "gaia-site-icon",
+  "version": "0.0.0",
+  "description": "A web component to show an icon for an application.",
```

Comment:
I'll fix it.
Do you want to be credited in the author field? If so which email address?